parcels/field.py

```diff
@@ -198,6 +202,7 @@ def from_netcdf(cls, filenames, variable, dimensions, indices=None, grid=None,
                     grid = CurvilinearSGrid(lon, lat, depth, time, time_origin=time_origin, mesh=mesh)
             grid.timeslices = timeslices
             kwargs['dataFiles'] = dataFiles
+            kwargs['netcdf_engine'] = netcdf_engine
```

Comment:
why is this in `if grid is None` and not later with other kwargs (at the end of the function)